slack/web/base_client.py

```diff
@@ -230,3 +232,12 @@ def _get_user_agent():
         system_info = "{0}/{1}".format(platform.system(), platform.release())
         user_agent_string = " ".join([python_version, client, system_info])
         return user_agent_string
+
+    @staticmethod
+    def validate_slack_signature(
```

Comment:
I agree that we don't need unit tests for this function. However, before we hit merge I would love to update this method with a docstring that explains where they can find the expected arguments. Or at least a link to the docs: https://api.slack.com/docs/verifying-requests-from-slack